test/issue674-concurrent-dub.sh

```diff
@@ -15,5 +15,4 @@ sleep 0.5
 cd ${TMPDIR} && $DUB fetch --cache=local bloom &
 pid2=$!
 wait $pid1
-wait $pid2
-[ -d ${TMPDIR}/bloom* ]
```

Comment:
Isn't this nullifying the test? Why was this necessary?

Reply:
Fixed